### PR TITLE
Chromeless UI Bugs

### PR DESCRIPTION
**Pull request by hghazzi**
- Error description displays for small players, padding adjusted so it
does not overlap with error title. Item description still stays hidden.
Affects with and without controls.

- Found another bug: Error display icon container now centered in
player for small players with controls.

- Poster image no longer show in controlbar only mode.

- Next Up container now hidden when controls are disabled.

JW7-4059

### CODE REVIEW

File: src/css/flags/controls-disabled.less [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .jw-flag-controls-disabled {
-    .jw-controls {
+    .jw-controls, .jw-nextup-container {
         visibility: hidden;
+        opacity: 0;
```
Comment (hghazzi) [inline src/css/flags/controls-disabled.less]: added to override opacity for `jw-nextup-container`

File: src/css/flags/time-slider-above.less [hunk 2]
```diff
@@ -27,6 +21,12 @@
       }
   }
 
+  &.jw-state-idle:not(.jw-flag-cast-available), &.jw-state-error {
+    .jw-display {
+      padding: 0;
+    }
+  }
+
```
Comment (hghazzi) [inline src/css/flags/time-slider-above.less]: Moved to increase specificity for the existing case and for error case. Ensures that the error display icon stays centered

File: src/css/imports/title.less [hunk 3]
```diff
@@ -48,7 +47,5 @@
 
   .jw-title-secondary {
     display: none;
-    margin-top: 5px;
```
Comment (hghazzi) [inline src/css/imports/title.less]: no longer needed since padding is adjusted in `states.less`

File: src/css/states/states.less [hunk 4]
```diff
@@ -97,8 +97,14 @@
...
 body .jwplayer.jw-state-error {
-  .jw-title .jw-title-primary {
-    white-space: normal;
+  .jw-title {
+    .jw-title-primary {
+      white-space: normal;
+    }
+    .jw-title-secondary {
+      display: block;
+      padding-top: 0.75em;
```
Comment (hghazzi) [inline src/css/states/states.less]: displays description during error state. `padding-top: 0.75em;` resets padding for `.jw-title-secondary` that was `padding: 0.75em 0;`

File: src/css/states/states.less [hunk 5]
```diff
@@ -163,10 +169,10 @@ body .jwplayer.jw-state-error {
...
-.jw-state-idle,
+.jw-state-idle:not(.jw-flag-audio-player),
 .jwplayer.jw-state-complete:not(.jw-flag-casting):not(.jw-flag-audio-player),
-body .jw-error,
-body .jwplayer.jw-state-error {
+body .jw-error:not(.jw-flag-audio-player),
+body .jwplayer.jw-state-error:not(.jw-flag-audio-player) {
   .jw-preview {
     display: block;
   }
```
Comment (hghazzi) [inline src/css/states/states.less]: Probably a better (and obvious) way of doing this, but couldn't figure it out. Only show preview for non controlbar only players

File: src/css/flags/controls-disabled.less (hunk 1 above)
Comment (robwalch) [inline src/css/flags/controls-disabled.less]: opacity is irrelevant when visibility is hidden. Why override it?

File: src/css/imports/title.less (hunk 3 above)
Comment (robwalch) [inline src/css/imports/title.less]: Setting it here set's it once and prevents it from changing when the state changes which is a performance improvement.

Comment (robwalch) [inline src/css/imports/title.less]: This also removes the padding above item descriptions in the idle state. you're now only setting it in the error state.

File: src/css/states/states.less (hunk 5 above)
Comment (robwalch) [inline src/css/states/states.less]: Yeah, don't change these rules. Add a new more specific rule for jw-flag-audio-player that set's jw-preview display to none.

File: src/css/flags/time-slider-above.less (hunk 2 above)
Comment (robwalch) [inline src/css/flags/time-slider-above.less]: 👍

File: src/css/flags/controls-disabled.less (hunk 1 above)
Comment (robwalch) [inline src/css/flags/controls-disabled.less]: Make sure only visibility: hidden is set for the problem cases:

".jw-flag-controls-disabled" makes the controls container hidden
but this is overriden by visible changes in 
".jw-nextup-container-visible" and ".jw-nextup-sticky .jw-nextup-close"

File: src/css/imports/title.less (hunk 3 above)
Comment (robwalch) [inline src/css/imports/title.less]: Undo and apply margin in error state / audio-player

File: src/css/states/states.less [hunk 6]
```diff
@@ -172,6 +179,34 @@ body .jwplayer.jw-state-error {
...
 }
 
+.jw-state-idle,
+.jwplayer.jw-state-complete:not(.jw-flag-casting),
+body .jwplayer.jw-state-error {
+  &.jw-flag-audio-player {
```
Comment (robwalch) [inline src/css/states/states.less]: There is only one state (error) that needs fixing. idle and complete are fine and should not be given new 2. 

> Poster shwon in controlbar-only mode
`body .jwplayer.jw-state-error .jw-preview { display: block; }`

File: src/css/states/states.less [hunk 7]
```diff
@@ -172,6 +179,34 @@ body .jwplayer.jw-state-error {
...
+    }
+
+    .jw-title-secondary {
+      width: auto;
+      display: inline-block;
+      padding-left: 0;
+    }
+  }
+}
+
```
Comment (robwalch) [inline src/css/states/states.less]: state-error is the issue, not jw-error

File: src/css/imports/title.less [hunk 8]
```diff
@@ -32,7 +31,7 @@
     margin-top: -.5em;
 }
 
-.jw-flag-small-player {
+.jw-flag-small-player:not(.jw-flag-audio-player) {
```
Comment (robwalch) [inline src/css/imports/title.less]: Please remove the not. It makes this rule more specific which can introduce regressions in other areas

**Pull request merged**